code/datums/uplink_item.dm

```diff
@@ -1227,8 +1207,7 @@ var/list/uplink_items = list()
 	name = "Macrobomb Implant"
 	desc = "An implant injected into the body, and later activated either manually or automatically upon death. Upon death, releases a massive explosion that will wipe out everything nearby."
 	item = /obj/item/weapon/storage/box/syndie_kit/imp_macrobomb
-	cost = 20
-	gamemodes = list(/datum/game_mode/nuclear)
+	cost = 16
```

Comment:
Waste of TC for nuke ops, keep it non nuke op only.
The whole point of macrobomb isn't to be a saving but to cause a lot of damage on death (i.e. die a glorious death objective.)

Shouldn't be reduced imo.